config/boards/sk-am68.csc

```diff
@@ -2,7 +2,7 @@
 
 BOARD_NAME="SK-AM68"
 BOARDFAMILY="k3"
-BOARD_MAINTAINER="3V3RYONE"
+BOARD_MAINTAINER=""
```

Comment:
I can take over as board maintainer. 3V3RYONE is an intern here at TI, so his work and this board is supported by us.